lib/GNUMakefilePyGeNN

```diff
@@ -0,0 +1,136 @@
+##--------------------------------------------------------------------------
+##   Author: Thomas Nowotny
+##           James Knight
+##           Anton Komissarov
+##
+##   Institute: Center for Computational Neuroscience and Robotics
+##              University of Sussex
+##              Falmer, Brighton BN1 9QJ, UK
+##
+##   email to:  T.Nowotny@sussex.ac.uk
+##
+##   initial version: 2018-05-14
+##
+##--------------------------------------------------------------------------
+
+SWIG                               :=swig -c++ -python
+SWIG_MODULE_NAME                   :=pygenn
+SWIG_PATH                          :=$(GENN_PATH)/lib/swig
+PYGENN_PATH                        :=$(GENN_PATH)/lib/pygenn
+PYGENN_SWIG_FILE_PATH              :=$(GENN_PATH)/lib/$(SWIG_MODULE_NAME).i
+PYTHON_INCLUDE                     ?=$(shell python-config --includes)
```

Comment:
For me, this returns: ``-I/usr/include/python2.7 -I/usr/include/x86_64-linux-gnu/python2.7`` so when you do ``-I $(PYTHON_INCLUDE)`` lower down it fails

Reply:
(it builds if I override it to ``/usr/include/python2.7`` from the command line though)

Reply:
Right... Python 2.7 is still the standard on many systems, isn't it?

Reply:
Oh, so the issue is not the version, but the second include? I also have two, but they are the same on my system... I guess, taking only the first one should solve this.

Reply:
Hah indeed I do add `-I`... I wonder why it works :thinking:

Reply:
By the way I cannot guarantee that Python 2.7 will work...

Reply:
I still don't really think restricting stuff to Python 3 is a legitimate option - a **lot** of people still use 2.7

Reply:
I will test it with 2.7.